Makefile.third

```diff
@@ -169,17 +169,10 @@ $(LUAJIT) $(LUAJIT_LIB) $(LUAJIT_JIT): $(THIRDPARTY_DIR)/luajit/CMakeLists.txt
 		-DCROSS="$(strip $(CCACHE) $(CHOST))-" \
 		$(CURDIR)/$(THIRDPARTY_DIR)/luajit && \
 		$(CMAKE_MAKE_PROGRAM) $(CMAKE_MAKE_PROGRAM_FLAGS)
-ifdef WIN32
-	cp -fL $(LUAJIT_DIR)/src/$(notdir $(LUAJIT_LIB)) $(LUAJIT_LIB)
-endif
-ifdef DARWIN
+ifdef USE_LUAJIT_LIB
 	cp -fL $(LUAJIT_DIR)/src/$(notdir $(LUAJIT_LIB)) $(LUAJIT_LIB)
 endif
-ifdef ANDROID
-	cp -fL $(LUAJIT_DIR)/src/$(notdir $(LUAJIT_LIB)) $(LUAJIT_LIB)
-else
```

Comment:
What about this else?

Reply:
this one already includes `ifdef DARWIN` right?

Reply:
But it said

```sh
if ANDROID
cp file x
else
cp file y
```

Now it's saying

```sh
if USE_LUAJIT_LIB
cp file x
endif
#ALWAYS
copy file y
```

Unless I'm misreading it, the functional equivalent should be something like `ifndef ANDROID`.

Reply:
Oh, sorry, good catch. You are right, I was only lookig at `DARWIN` :P

Will think about how to update to test to catch this going forward.